MaxMind.GeoIP2/Responses/AnonymousPlusResponse.cs

```diff
@@ -0,0 +1,152 @@
+﻿#region
+
+using MaxMind.Db;
+using System;
+using System.Text.Json.Serialization;
+
+#endregion
+
+namespace MaxMind.GeoIP2.Responses
+{
+    /// <summary>
+    ///     This class represents the GeoIP Anonymous Plus response.
+    /// </summary>
+    public class AnonymousPlusResponse : AnonymousIPResponse
+    {
+        /// <summary>
+        /// Construct AnonymousPlusResponse model
+        /// </summary>
+        public AnonymousPlusResponse()
+        {
+        }
+
+#if NET6_0_OR_GREATER
+        /// <summary>
+        /// Construct AnonymousPlusResponse model
+        /// </summary>
+        /// <param name="anonymizerConfidence"></param>
+        /// <param name="isAnonymous"></param>
+        /// <param name="isAnonymousVpn"></param>
+        /// <param name="isHostingProvider"></param>
+        /// <param name="isPublicProxy"></param>
+        /// <param name="isResidentialProxy"></param>
+        /// <param name="isTorExitNode"></param>
+        /// <param name="ipAddress"></param>
+        /// <param name="network"></param>
+        /// <param name="networkLastSeen"></param>
+        /// <param name="providerName"></param>
+        [Constructor]
+        public AnonymousPlusResponse(
+            [Parameter("anonymizer_confidence")] int anonymizerConfidence,
+            [Parameter("is_anonymous")] bool isAnonymous,
+            [Parameter("is_anonymous_vpn")] bool isAnonymousVpn,
+            [Parameter("is_hosting_provider")] bool isHostingProvider,
+            [Parameter("is_public_proxy")] bool isPublicProxy,
+            [Parameter("is_residential_proxy")] bool isResidentialProxy,
+            [Parameter("is_tor_exit_node")] bool isTorExitNode,
+            [Inject("ip_address")] string? ipAddress,
+            [Network] Network? network = null,
+            [Parameter("network_last_seen")] string? networkLastSeen = null,
+            [Parameter("provider_name")] string? providerName = null
+        ) : this(anonymizerConfidence, isAnonymous, isAnonymousVpn, isHostingProvider, isPublicProxy,
+                isResidentialProxy, isTorExitNode, ipAddress, network,
+                networkLastSeen == null ? null : DateOnly.Parse(networkLastSeen),
```

Comment:
For anyone looking at this note I haven't reviewed this PR, this is just a flyby comment...

This probably doesn't matter realistically (since for this to happen something would have had to gone very wrong in the API), but quickly looking at the other response models it looks like this might be the only one that can throw an exception in the constructor. I'm not sure if it would be more idiomatic to use `DateOnly.TryParse` here instead.

Reply:
There would already be an exception when deserializing to any of these models if the data in the database did not match what was expected in other ways, e.g., the database has a string and the model is expecting an int. Although that exception would technically not be in the constructor, it would be effectively the same from a user's perspective.